index.html

```diff
@@ -3277,31 +3277,58 @@ <h3>Reserved Extension Points</h3>
 see Section <a href="#types"></a>.
         </p>
 
-        <p class="issue" title="Reserved properties under debate">
-The following extension point properties are under consideration for being
-marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
-`evidence`, `renderMethod`, `refreshService`, and
-`confirmationMethod/confidenceMethod`.
-        </p>
-
         <table class="simple">
           <thead>
             <tr>
               <th>Reserved&nbsp;Property</th>
               <th>Description</th>
             </tr>
           </thead>
-
           <tbody>
             <tr>
+              <td>`confirmationMethod` / `confidenceMethod`</td>
+              <td>
+A property used for specifying how a <a>verifier</a> can confirm, or
+gain more confidence in, the presentation of the credential being associated
+with a particular <a>subject</a>.
+
+                <p class="issue" title="Bikeshedding confirmation/confidence">
+The name of this property is currently being debated by the Working Group.
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <td>`evidence`</td>
```

Comment:
need to reserve this term URL here, as well:

https://www.w3.org/2018/credentials#evidence

Reply:
Correct. I plan to make a revision of the vocabulary document, as well as the yml2vocab tool (so that experimental terms are presented separately) if and when these PR are merged.